agent/src/agent/data_extractor/observium/observium.py

```diff
@@ -31,18 +31,21 @@ def _get(pipeline_: Pipeline):
 def _create_metrics(data: dict, pipeline_: Pipeline) -> list:
     metrics = []
     # these values must be outside the for loop for optimization purposes
-    fields = field.build_fields(pipeline_.dimension_configurations)
-    value_paths = pipeline_.value_paths
-    value_paths = dict(zip(value_paths, value_paths))
+    fields_dims = field.build_fields(pipeline_.dimension_configurations)
+    fields_meas = field.build_fields(pipeline_.values_configurations)
+    fields_tags = field.build_fields(pipeline_.tags_configurations)
```

Comment:
could you please also generate 10 000 records of observium data in the dummy_destination and test how fast it will process them? There were some performance issues in the first release, so want to be sure it's fine. Don't need to push it, just check locally

Reply:
Performed measurement on entire extract_metrics functiion like this:
```
    t1_start = perf_counter()
    metrics = data_extractor.observium.extract_metrics(pipeline_)
    t1_stop = perf_counter()
```
objects in data - time
150000 - 6.814 sec
30000 - 1.343 sec
9000 - 0.363 sec
900 - 0.048 sec

Reply:
nice, thanks